src/aiidalab_qe/app/utils/search_jobs.py

```diff
@@ -183,57 +205,67 @@ def setup_widgets(self):
         self.time_start.observe(self.apply_filters, names="value")
         self.time_end.observe(self.apply_filters, names="value")
         self.job_state_dropdown.observe(self.apply_filters, names="value")
-        self.label_search_field.observe(self.apply_filters, names="value")
-
-        self.filters_layout = ipw.VBox(
-            [
-                ipw.HTML("<h3>Search & filter calculations:</h3>"),
+        display_options = ipw.VBox(
+            children=[
+                ipw.HTML("<h4>Display options:</h4>"),
                 ipw.VBox(
                     [
-                        self.job_state_dropdown,
-                        self.time_box,
-                        ipw.HBox(
-                            [
-                                self.label_search_description,
-                                self.label_search_field,
-                            ]
-                        ),
-                        ipw.VBox(
-                            [self.properties_filter_description, self.properties_box]
-                        ),
-                        #   self.apply_filters_btn,
+                        self.toggle_time_format,
+                        self.toggle_id_format,
+                        # self.toggle_multi_selection,
                     ]
                 ),
-                ipw.HTML("<h3>Display options:</h3>"),
+            ],
+            layout=ipw.Layout(
+                border="1px solid #ddd",
+                padding="0px",
+                margin="0px, 0px, 100px, 0px",
+                background_color="#f9f9f9",
+                border_radius="5px",
+                box_shadow="0 2px 5px rgba(0, 0, 0, 0.1)",
+            ),
+        )
+        filters = ipw.VBox(
+            children=[
+                ipw.HTML("<h4>Filters:</h4>"),
                 ipw.VBox(
                     [
-                        self.toggle_description_checkbox,
-                        self.toggle_time_format,
-                        self.toggle_id_format,
+                        self.job_state_dropdown,
+                        self.time_box,
+                        ipw.VBox(
+                            [self.properties_filter_description, self.properties_box],
+                            layout=ipw.Layout(
+                                border="1px solid #ddd", padding="5px", margin="5px"
```

Comment:
```suggestion
                                border="1px solid #ddd", 
                                padding="5px", 
                                margin="5px",
```

I promise I'm not being picky 😅 This line by line format makes git maintenance easier.